app/views/pregnancies/edit.html.erb

```diff
@@ -16,7 +16,7 @@
       <%= render partial: 'pregnancies/patient_information', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/change_log', locals: { f: f, pregnancy: @pregnancy } %>
       <%= render partial: 'pregnancies/call_log', locals: { pregnancy: @pregnancy } %>
-      <% end %>
+<% end %>
```

Comment:
(Which is to say, sorry! I'm sure you looked at that and were like 'what in the...')